.github/workflows/tests.yml

```diff
@@ -0,0 +1,59 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    env:     
+      FLOW_TARGET_VERSION: 6.3
+      FLOW_CONTEXT: Testing
+      FLOW_FOLDER: ../flow-base-distribution
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Update Composer
+      run: |
+        sudo composer self-update
+        composer --version
+        
+    # Directory permissions for .composer are wrong, so we remove the complete directory
+    # https://github.com/actions/virtual-environments/issues/824
+    - name: Delete .composer directory
+      run: |
+        sudo rm -rf ~/.composer
+
+    - name: Cache dependencies
+      uses: actions/cache@v1
+      with:
+        path: ~/.composer/cache
+        key: dependencies-composer-${{ hashFiles('composer.json') }}
+
+    - name: Prepare Flow distribution
+      run: |
+        git clone https://github.com/neos/flow-base-distribution.git -b ${FLOW_TARGET_VERSION} ${FLOW_FOLDER}
+        cd ${FLOW_FOLDER} 
+        composer require --no-update --no-interaction flowpack/restapi:dev-master
```

Comment:
This could be replaced with a script that inserts a `"repository"` section with the path to the checkout directory to the composer.json

If we can somehow use a variable instead of the hardcoded checkout path (repository name), then this would be a general template for all Flow packages :)